packages/README.md

```diff
@@ -89,3 +89,12 @@ Changes:
 If you accept, Lerna will create git tags, publish those to github, then push your packages to npm.
 
 🎉 You have a published package!
+
+### Publishing a single package
+
+Sometimes, its helpful to release a singular package. This can be done directly through npm. Be sure versions and builds are correct.
+
+-   Bump the version in the package's package.json as well as its CHANGELOG file.
```

Comment:
Do you use `npm version` to bump this, or is it best just to update it manually?

Reply:
I think manual is best. From the docs, it appears there's some additional things going on:

>If run in a git repo, it will also create a version commit and tag